devTools/regionsImporter/update.ts

```diff
@@ -0,0 +1,142 @@
+import { readFile, writeFile } from "fs/promises"
+import { execFileSync } from "child_process"
+import { createHash } from "crypto"
+import { parse } from "papaparse"
+import prettier from "prettier"
+import fetch from "node-fetch"
+import _ from "lodash"
+
+const ETL_REGIONS_URL =
+        "https://catalog-staging.ourworldindata.org/grapher/regions/latest/regions/regions.csv",
```

Comment:
I'm not up-to-date with the latest decisions here, but why are we not using `regions.yml` instead?

Reply:
The decision was to add a grapher step to the ETL to generate the region data at a fixed url (whereas the regions.yml file would always be at a path with a potentially changing date-string in it). See [etl/#1027](https://github.com/owid/etl/pull/1027)